tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -363,6 +363,26 @@ public static class Tags
         /// </summary>
         public const string AzureAppServicesSubscriptionId = "aas.subscription.id";
 
+        /// <summary>
+        /// The type of trigger for an azure function
+        /// </summary>
+        internal const string AzureFunctionTriggerType = "aas.function.trigger";
+
+        /// <summary>
+        /// The UI name of the azure function
+        /// </summary>
+        internal const string AzureFunctionName = "aas.function.name";
+
+        /// <summary>
+        /// The UI name of the azure function
```

Comment:
nit: This comment matches the one above. Is this supposed to be the full name of the azure function?